packages/@okta/vuepress-site/docs/release-notes/2020-12-0/index.md

```diff
@@ -13,7 +13,7 @@ title: Okta API Products Release Notes
 | [One Time Use Refresh Token is now in Early Access (EA)](#one-time-use-refresh-token-is-now-in-early-access-ea)      | December 9, 2020         |
 | [Enhancements to Apps API for Idp Initiated Logins](#enhancements-to-apps-api-for-idp-initiated-logins) | December 9, 2020         |
 | [Enhancements to Apps API for SAML Apps](#enhancements-to-apps-api-for-saml-apps) | December 9, 2020         |
-| [Group object `source` property is now GA in Production](#group-object-source-property-is-now-ga-in-production) | December 9, 2020        |
+| [Group object source property is now GA in Production](#group-object-source-property-is-now-ga-in-production) | December 9, 2020        |
```

Comment:
This should read something closer to "Groups API extended search in GA in Preview", right?